spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+RSpec.describe UsersController, type: :controller do
+  let!(:user) { create(:user) }
+
+  before { controller.stub(:current_user) { user } }
+
+  describe 'GET #edit' do
+    before { get :edit, params: { id: user.id } }
+
+    it 'render edit' do
+      expect(response).to render_template(:edit)
+    end
+  end
+
+  describe 'PUT #update' do
+    context 'when good update' do
+      let(:password) { 'Password1' }
+      let(:password_confirmation) { password }
+
+      before do
```

Comment:
move to global before for current describe 'PUT #update'

Reply:
refactor